packages/storybook-vue/src/stories/CustomRadioButton/CustomRadioButtonStates.stories.tsx

```diff
@@ -0,0 +1,101 @@
+import type { Meta, StoryFn } from '@storybook/vue3';
+import { CustomRadioButton } from '@utrecht/component-library-vue';
+import { argTypes } from './argTypes';
+
+export default {
+  component: CustomRadioButton,
+  id: 'vue-custom-radio-button-states',
+  title: 'Vue.js Component/Custom Radio Button/States',
+  argTypes: argTypes,
+} as Meta<typeof CustomRadioButton>;
+
+const Template: StoryFn<typeof CustomRadioButton> = (args) => ({
+  components: { CustomRadioButton },
+  setup() {
+    return { args };
+  },
+  template: '<CustomRadioButton v-bind="args" />',
+});
+
+export const Checked = Template.bind({});
+Checked.args = {
+  checked: true,
+  value: 'checked',
+};
+Checked.parameters = {
+  docs: {
+    source: {
+      code: `<CustomRadioButton :checked="true" value="checked"/>`,
```

Comment:
I wonder if these `value` attributes in the examples for CustomRadioButton might be more confusing than helpful. Especially this example, folks might think `value="checked"` is just as essential to copy as `:checked="true"`.

Reply:
I will remove them